api/envoy/config/filter/http/grpc_http1_reverse_bridge/v2alpha1/config.proto

```diff
@@ -24,4 +24,5 @@ message FilterConfig {
   // hide the gRPC semantics from the upstream, allowing it to receive and respond with a
   // simple binary encoded protobuf.
   bool withhold_grpc_frames = 2;
+  bool enabled = 3;
```

Comment:
Let's change the sense of this field such that the default value (false) enables the bridge. That is, let's call this flag `disabled`.

This way, an existing per-route config for this filter will not suddenly become disabled when Envoy is upgraded. Further, if an operator wants to set `content_type` or `withhold_grpc_frames`, they won't also have to specifically enable the filter.

Reply:
> Let's change the sense of this field such that the default value (false) enables the bridge. That is, let's call this flag `disabled`.
> 
> This way, an existing per-route config for this filter will not suddenly become disabled when Envoy is upgraded. Further, if an operator wants to set `content_type` or `withhold_grpc_frames`, they won't also have to specifically enable the filter.

Changed: https://github.com/envoyproxy/envoy/pull/8145/commits/18997a7a6b53ab65d642e24b22327f843a6e152a